_includes/share-buttons.html

```diff
@@ -26,6 +26,8 @@ <h3 class="">Embedded Events</h3>
     </p>
 </div>
 
+{% include notification-banner.html %}
```

Comment:
Given that this is no longer a configurable notification banner and rather a sidebar item, it would be best to incorporate the html directly here to match the other sidebar items.

Reply:
Hey @gminn 
So just to make sure, we aren't looking to make it configurable any more, just plain HTML, right?